webextension/privileged/ui/CloudDownloadsView.jsm

```diff
@@ -137,9 +132,14 @@ var CloudDownloadsView = {
         fragment.appendChild(moveDownloadItem);
       }
       aPopupMenu.insertBefore(fragment, menuItem.nextSibling);
-      aPopupMenu.addEventListener("click", this);
+      aPopupMenu.addEventListener("command", this);
+      aPopupMenu.addEventListener("popuphidden", this);
```

Comment:
@gijsk would like to use this event to capture when downloadContextMenu is hidden, to hide 'moveDownload' menuitem in downloadContextMenu. This is needed to avoid showing moveDownload option in context menu from Library -> Downloads -> Right click download item. 
Is popuphidden the right event to handle this use case?